### ISSUE TITLE
Mobile first design 

### ISSUE THREAD

**Issue by ShaneMoyo** [Martsyalis/portfolio#6]
I may have missed something but I did not notice any difference when scaling the screen up and down. Maybe you could make it a bit more responsive to the changing screen size.

**Comment by Martsyalis** [comment]
yeah im so mobile first I didn't even adjust it for the full screen yet haha, the truth is I don't know how I want it to look yet. but for practice sake i should try to add break points
